Makefile

```diff
@@ -64,7 +64,7 @@ cni-plugin-lint:
 	cd control-plane/cni; golangci-lint run -c ../../.golangci.yml
 
 ctrl-generate: get-controller-gen ## Run CRD code generation.
-	cd control-plane; $(CONTROLLER_GEN) object:headerFile="build-support/controller/boilerplate.go.txt" paths="./..."
+	cd control-plane; $(CONTROLLER_GEN) object paths="./..."
```

Comment:
This empty header file was originally needed to satisfy the Controller Gen CLI. We never had anything in it.

Reply:
CC @lkysow because he had to add the empty file in the first place.